packages/react-table/src/components/Table/examples/TableCellWidth.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Table, TableHeader, TableBody, classNames, cellWidth, Visibility, TableProps } from '@patternfly/react-table';
+
+interface Repository {
+  name: string;
+  branches: string;
+  prs: string;
+  workspaces: string;
+  lastCommit: string;
+}
+
+export const TableCellWidth: React.FunctionComponent = () => {
+  // In real usage, this data would come from some external source like an API via props.
+  const repositories: Repository[] = [
+    {
+      name: 'one - 1',
+      branches: 'two - 1 (visible only on md)',
+      prs: 'three - 1 (hidden only on md)',
+      workspaces: 'four - 1 (hidden on xs)',
+      lastCommit: 'five - 1'
+    },
+    {
+      name: 'one - 2',
+      branches: 'two - 2 (visible only on md)',
+      prs: 'three - 2 (hidden only on md)',
+      workspaces: 'four - 2 (hidden on xs)',
+      lastCommit: 'five - 2'
+    }
+  ];
+
+  const columns: TableProps['cells'] = [
+    { title: 'Header cell', transforms: [cellWidth(10)] },
+    {
+      title: 'Branches (visible only on md)',
+      columnTransforms: [
+        classNames(Visibility.hidden, Visibility.visibleOnMd, Visibility.hiddenOnLg, Visibility.visibleOn2Xl)
```

Comment:
Good catch, thanks

Reply:
@seanforyou23 I'm not sure I follow what would cause that error. I'm passing 4 defined strings into `classNames` here

Reply:
I thought it was strange too, I hadn't seen the `IVisibility` type previously so still looking. hmm I'll try rebuilding locally in case it's just an issue on my machine

<img width="834" alt="Screen Shot 2021-11-23 at 11 55 31 AM" src="https://user-images.githubusercontent.com/5942899/143069158-11d3e289-4371-481a-b6bb-ead88ee6900b.png">

Reply:
Resolving based on slack conversation.